static/app/views/app/index.tsx

```diff
@@ -23,14 +23,17 @@ import {onRenderCallback} from 'sentry/utils/performanceForSentry';
 import useApi from 'sentry/utils/useApi';
 import {useColorscheme} from 'sentry/utils/useColorscheme';
 import {useHotkeys} from 'sentry/utils/useHotkeys';
+import type {InstallWizardProps} from 'sentry/views/admin/installWizard';
 
 import SystemAlerts from './systemAlerts';
 
 type Props = {
   children: React.ReactNode;
 } & RouteComponentProps<{orgId?: string}, {}>;
 
-const InstallWizard = lazy(() => import('sentry/views/admin/installWizard'));
+const InstallWizard: React.FC<InstallWizardProps> = lazy(
+  () => import('sentry/views/admin/installWizard')
+);
```

Comment:
This is annoying, why does it need to be typed now?

Reply:
for some reason it's having trouble getting the props out of the async component class